openmdao/utils/unit_library.ini

```
@@ -196,6 +196,7 @@ rps: 2*pi*rad/s, RPS
 cal: 4.184*J, thermochemical calorie
 cali: 4.1868*J, international calorie
 Btu: 1055.05585262*J, British thermal unit
+MMBtu: 1055.05585262E3*J, one thousand Btu

```

interesting that MMMBtu will also be valid.

---

Thanks for moving on this so quickly! I think this line should be:

```suggestion
MMBtu: 1055.05585262E6*J, one million Btu

```